.github/workflows/build.yml

```diff
@@ -303,11 +317,34 @@ jobs:
           as well as the [downloads section](https://ppsspp.org/downloads.html)
           for other platforms.
 
+    - name: Create iOS release
+      if: startsWith(github.ref, 'refs/tags/') && runner.os == 'macOS' && matrix.extra == 'ios'
+      run: |
+        cd ppsspp || exit 1
+        mv PPSSPP.ipa PPSSPPSDL-iOS-${GITHUB_REF##*/}.ipa || exit 1
+
+    - name: Upload iOS release
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/') && runner.os == 'macOS' && matrix.extra == 'ios'
+      with:
+        files: ppsspp/*.ipa
+        body: >
+          PPSSPP is a cross-platform PSP emulator.
+          Visit PPSSPP [official website](https://ppsspp.org)
+          for a [full changelog](https://ppsspp.org/index.html#news)
+          as well as the [downloads section](https://ppsspp.org/downloads.html)
+          for other platforms.
```

Comment:
Does this work fine if it tries to create both releases?  Assuming it does, would really prefer to avoid duplicating the whole thing here.

It might be best to use a local action and encapsulate the duplication there.  See:
https://stackoverflow.com/questions/60544181/how-to-use-snippets-in-github-action-workflow-file-to-avoid-duplicates

-[Unknown]

Reply:
i'm not sure whether the release and upload function worked or not, i don't know how to trigger that part of the script on my fork.
How can i test the release part of the script?

Reply:
I think you can just create a release on your fork, number it anything you want.

-[Unknown]

Reply:
btw, what you meant to use local action is for the `uses: softprops/action-gh-release@v1` right?
i'm not sure what that `action-gh-release@v1` actually do, are they accessible? to view the script that is, as i don't know what kinda local action i should create to replace it

Reply:
hmm.. may be i should just modify the `files:` on the "Upload macOS release" to include *.ipa files like this:
```
files: >
      ppsspp/*.zip
      ppsspp/*.ipa
```